ui/src/app/pages/access/compliance-training-module-card-title.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { useEffect } from 'react';
+
+import { AccessModule, Profile } from 'generated/fetch';
+
+import { AoU } from 'app/components/text-wrappers';
+import { profileApi } from 'app/services/swagger-fetch-clients';
+import { AccessTierShortNames } from 'app/utils/access-tiers';
+import {
+  getAccessModuleStatusByName,
+  isCompliant,
+  isExpiringOrExpired,
+} from 'app/utils/access-utils';
+
+export interface ComplianceTrainingModuleCardProps {
+  tier: AccessTierShortNames;
+  profile: Profile;
+}
+
+export const ComplianceTrainingModuleCardTitle = ({
+  tier,
+  profile,
+}: ComplianceTrainingModuleCardProps) => {
+  const [useAbsorb, setUseAbsorb] = React.useState(false);
+  useEffect(() => {
+    profileApi().useAbsorb().then(setUseAbsorb);
+  }, []);
+
+  const { accessModule, trainingTitle, courseTitle } =
+    tier == AccessTierShortNames.Registered
+      ? {
+          accessModule: AccessModule.COMPLIANCE_TRAINING,
+          trainingTitle: 'Responsible Conduct of Research Training',
+          courseTitle: 'Responsible Conduct of Research',
```

Comment:
This is different text than before. See [this comment](https://pmi-engteam.slack.com/archives/C02MWP2RN5P/p1696886542210769?thread_ts=1696878015.979769&cid=C02MWP2RN5P).